ckanext/ontario_theme/templates/internal/package/snippets/resource_edit_form.html

```diff
@@ -0,0 +1,9 @@
+{% ckan_extends %}
+
+{% block save_button %}
+  <button class="ontario-button ontario-button--primary"
+          name="save"
+          type="submit">
+    {% block save_button_text %}Continue{% endblock %}
```

Comment:
```suggestion
    {% block save_button_text %}
      Continue
    {% endblock save_button_text %}
```

Reply:
This is fine, but fyi it is not flagged by `djlint`.

Reply:
@KatiRG it wouldn't be flagged with an unconfigured djLint. It would be flagged with our pre-commit hooks which is configured for our repo.